src/Form/EntityDeleteLogSeetingsForm.php

```diff
@@ -76,7 +76,7 @@ public function buildForm(array $form, FormStateInterface $form_state) {
     // Add some helpful links.
     $help_links = [
       Link::fromTextAndUrl('View the Entity Delete Log README',
-          Url::fromUserInput('/' . drupal_get_path('module', 'entity_delete_log') . '/README.txt')),
+          Url::fromUserInput('/' . \Drupal::service('extension.path.resolver')->getPath('module', 'entity_delete_log') . '/README.txt')),
```

Comment:
@shashank-naik Thanks for working on this. Can we use DI here instead of static call.

Reply:
Yeah, it should be possible. I'll look into it.

Reply:
I've made this change. Please review when you can. I've also corrected a minor typo in the classname. Thanks!